app/Controllers/InviteCodeController.js

```diff
@@ -0,0 +1,44 @@
+const {
+  createInviteCode,
+  destroyInviteCode,
+  getInviteCodes,
+  validateInviteCode,
+} = require('../Services/InviteCodeProvider.js');
+const catchAsync = require('../utils/catchAsync');
+
+const addInviteCode = catchAsync(async (req, res) => {
+  const maxUses = req.query.maxUses || null;
```

Comment:
Data to save in a post request should be in the body.

Reply:
done